src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -179,19 +200,32 @@ public boolean deleteSubscription(String subscriptionName) throws AccessExceptio
         return subscriptions;
     }
 
+    private Subscription doEncryption(Subscription subscription) {
+        String password = subscription.getPassword();
+        try {
+            String encryptedPassword = EncryptionFormatter.addEncryptionParentheses(
+                    encryptor.encrypt(password));
+            subscription.setPassword(encryptedPassword);
+        } catch (EncryptorException e) {
+            LOGGER.error("", e);
```

Comment:
Question is, do we want to store it unencrypted if the encryption doesnt work for some reason? Maybe it should just not store it at all? Could be good to ask glue about this to see if this is an allowed solution or not. It feels like we shoulndt store the subscription at all if this happens and user should perhaps see this error in the gui while creating the subscription. 

Otherwise the user might think that everything is good while infact, their password is stored as plain text.

Reply:
I removed my own exception and use the error from the encryption lib. So there will be an error on subscription creation and notification of the type EncryptionOperationNotPossibleException. The most common problem for us might be issues with the secret.